Moderation-Policy.md

```diff
@@ -0,0 +1,48 @@
+## Moderation policy
+
+### Applicability
+
+By default, this policy applies to all repositories under the Node.js Github Organization and all Node.js Working Groups.
+
+Individual Working Groups and Top Level Projects chartered by the TSC may adopt an alternative Moderation Policy for any repository under their stewardship so long as:
+* The Moderation Policy is openly documented as part of the Working Group or Project Charter and;
+* Includes provisions for clearly and openly documenting Moderation actions taken.
+
+### Terms
+
+* "Collaborator" refers to any individual member or "outside collaborator" of the Node.js Github Organization.
+* "Post" refers to the content of any issue, pull request or comment.
+* "Moderate" refers to the act of modifying the content of or deleting any Post that is in obvious violation of the Node.js Code of Conduct.
+* "Ban" refers to the act of blocking an individual Github account from any further participation in the Node.js Github Organization.
+
+### Policy
+
+* All Posts are expected to respect the Node.js Code of Conduct.
+* Only Collaborators with commit rights to a given repository may Moderate Posts within that repository's issue tracker.
+* Collaborator's must not Moderate any Post authored by another Collaborator without first giving the author an opportunity of at least 24 hours to modify or remove the Post on their own.
```

Comment:
At first I thought this was a bad suggestion since all collaborators are expected to follow the CoC and the CoC should apply equally to all members, collaborators or not. Then I re-read the CoC and found that is has a lot of «suggestions» (like the use of personal pronouns) which I think does not warrant an edit by another collaborator without notice.